mutiny-core/src/lsp/lsps.rs

```diff
@@ -355,7 +344,7 @@ impl<S: MutinyStorage> Lsp for LspsClient<S> {
         // this is only here to keep both voltage + lsps logic symmetric
         if inbound_capacity_msat >= fee_request.amount_msat {
             return Ok(FeeResponse {
-                id: None,
+                id: String::from("deadbeef"),
```

Comment:
it's not actually :)

we aren't using the flow in this case. if they have liquidity we want to just generate a regular invoice and not use lsps.

so instead of returning some kind of Error here, to keep the flows symmetric it just returns nonsense response that won't get used

Reply:
but does have me wondering why we even have to enter the flow, let me review and remind myself

Reply:
seems like it can be refactored out, let me try

Reply:
ok, removed the fee check when user has enough inbound liquidity on lsps flow